src/xGadgetron/cGadgetron/gadgetron_data_containers.cpp

```diff
@@ -378,11 +378,13 @@ complex_float_t a, complex_float_t b)
 	int n = y.number();
 	ISMRMRD::Acquisition ax;
 	ISMRMRD::Acquisition ay;
+	ISMRMRD::Acquisition acq;
```

Comment:
updates here don't seem have anything to do with the title of this PR. Are these intentional?

Reply:
all updates are necessary for in-place algebra the way I implemented it: e.g. now that we want to overwrite acquisition data, we need to check first whether the acquisition to be overwritten is not to be ignored